extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -185,8 +185,8 @@ define('ext.wikia.adEngine.template.porvata', [
 
 		if (prebid && params.hbAdId) {
 			params.bid = prebid.getBidByAdId(params.hbAdId);
-			params.vastResponse = params.bid.vastContent || null;
-			params.vastUrl = params.bid.vastUrl;
+			params.vastResponse = params.bid && params.bid.vastContent ? params.bid.vastContent : null;
```

Comment:
How is it possible that there is `params.hbAdId` but there is no `prebid.getBidByAdId(params.hbAdId)`? Are you sure that this fix isn't hiding real problem?